### ISSUE TITLE
MS word: NVDA should announce if the text is in two columns

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3995]
**Reported by Sukil on 2014-03-19 10:42**
Currently, if a multicolumn text is opened in Word, NVDA doesn't announce this information. This maylead to skip important parts of text.
**Blocking #4547**

**Comment by bhavyashah** [comment]
While my usage of Microsoft Office (especially word) is restricted mostly to academics and I have had to do little with multi-column documents, an online course I was once sort of part of (Whizjuniors was its name, in case anyone was wondering which course) had working with multi-column documents as one of initial and basic parts of a word processor. This observation, coupled with the fact that this has been brought up on Github several times as well as probably on the NVDA Users list some times, seems to imply that this lack of support is adversely impacting many since it sounds like multi-column documents are an integral part of more advanced word processing or corporate environments. Taking the work-around shared by @jcsteh in https://github.com/nvaccess/nvda/issues/3995#issuecomment-155318573 into account, @Qchristensen, could you please enlighten us about the significance of the requested feature for an average NVDA user using Microsoft Word, so that we may better evaluate the importance of this ticket itself?

**Comment by jcsteh** [comment]
This is implemented now; see #6536. Closing.
